nephio-ansible-install/roles/clab/tasks/main.yaml

```diff
@@ -6,3 +6,60 @@
 - name: installing clab
   become: true
   shell: bash -c "$(wget -qO - https://get.containerlab.dev)"
+  when: validate_certs
+
+
+- name: Create clab installation directory
```

Comment:
This change is to add skip ssl/tls checks for all the curl commands and wget commands in the container lab installation script if the validate_certs is set to false.